README.md

```diff
@@ -603,6 +603,39 @@ $ tree .
 └── staged
 ```
 
+#### Adding a new root key
+
+Copy `staged/root.json` to the root box and generate a new root key on the root box:
+
+```bash
+$ tuf gen-key root
+$ tuf sign root.json
+```
+
+Copy `staged/root.json` from the root box to commit:
```

Comment:
nit: this means copy `staged/root.json` to the repository working directory AND then commit in that dir?

if so, maybe say `Copy `staged/root.json` from the root box and commit`